.gitignore

```diff
@@ -18,3 +18,5 @@ docs/_*/*
 docs/autodoc/
 build
 pyan/
+**.coverage
+**htmlcov*
```

Comment:
The changes to `.gitignore` definitely should go as a separate PR with motivation. It's clearly unrelated to the base58 code.